app/db/queries/application.py

```diff
@@ -113,9 +127,25 @@ def clone_single_section(section_id: str, new_round_id=None) -> Section:
     return clone
 
 
-def clone_single_form(form_id: str, new_section_id=None) -> Form:
+def _fix_cloned_default_pages(cloned_pages: list[Page]):
```

Comment:
would be nice for this to be part of the page clone itself, i Imagine the issue there is that you need all the page ids to match the original default to the new - hence it comes after?

Reply:
Yeah that's exactly it - I can't update a page with its next default ID until that next page has been made, so much simpler to just do after.